targets/android-studio

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+REQUIRES='xiwi gtk-extra'
```

Comment:
OpenJDK8 and other dependencies of Android Studio in turn depend on GTK+, hence the gtk-extra target.